lms/djangoapps/email_marketing/tasks.py

```diff
@@ -294,6 +300,97 @@ def _get_course_content(course_url, sailthru_client, email_config):
     return response
 
 
+def _get_or_create_sailthru_list_for_site(sailthru_client, site=None, default_list_name=None):
+    """
+    Get the user list name from cache if exists else create one and return the name,
+    callers of this function should perform the enabled check of email config.
+    :param: sailthru_client
+    :param: site
+    :param: default_list_name
+    :return: list name if exists or created else return None
+    """
+    if site and site.id != settings.SITE_ID:
+        list_name = site.domain.replace(".", "_") + "_user_list"
+    else:
+        list_name = default_list_name
+
+    sailthru_list = _get_or_create_sailthru_list(sailthru_client, list_name)
+    return list_name if sailthru_list else default_list_name
+
+
+def _get_or_create_sailthru_list(sailthru_client, list_name):
```

Comment:
So, here you can change the name to `_get_or_create_user_list`. Remember to update your comments with this as well